docs/shared/features/run-tasks.md

```diff
@@ -1,12 +1,91 @@
 # Run Tasks
 
-Monorepos can have hundreds or even thousands of projects, so being able to run actions against all (or some) of them is a key feature of a tool like Nx.
+In a monorepo setup, you don't just run tasks for a single project; you might have hundreds to manage. To help with this, Nx provides a powerful task runner that allows you to:
 
-## Types of Tasks
+- **speed up task execution** with caching
+- easily **run multiple targets** for multiple projects **in parallel**
+- define **task pipelines** to run tasks in the correct order
+- only run tasks for **projects affected by a given change**
 
-Nx tasks can be created from existing `package.json` scripts, [inferred from tooling configuration files](/concepts/inferred-tasks), or defined in a `project.json` file. Nx will merge all three sources together to determine the tasks for a particular project.
+## Defining Tasks
 
-Read the [Project Configuration docs](/reference/project-configuration) to see all the configuration options for a task.
+Nx tasks can be created from existing `package.json` scripts, [inferred from tooling configuration files](/concepts/inferred-tasks), or defined in a `project.json` file. Nx combines these three sources to determine the tasks for a particular project.
+
+{% tabs %}
+{% tab label="package.json" %}
+
+```json
+{
+  "name": "mylib",
+  "scripts": {
+    "test": "jest",
+    "build": "tsc -p tsconfig.lib.json"
```

Comment:
Put build first so that it matches the same order as the `project.json` tab (for people toggling back and forth to compare)

```suggestion
    "build": "tsc -p tsconfig.lib.json",
    "test": "jest"
```